JSDemos/Demos/Gauges/GaugeTitleCustomized/React/description.md

```diff
@@ -0,0 +1,11 @@
+This demo shows how to specify the CircularGauge title and display custom labels in the component's center. 
+
+The CircularGauge title usually contain general explanation about the data represented by the gauge. You can configure it with the properties of the [title](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/) configuration object:
```

Comment:
Same thing - we refer to an abstract title, not specific. The word "usually" is a hint that it's not one specific title, but we mean "any title". 

Anyway, the first sentence is a bit too obvious (we explain why people might want a title in a component). I would either simplify it, or maybe remove it altogether. (That's why I added two different suggestions.)

In the last sentence, I don't want to use both words "configure" and "configuration". 

```suggestion
A CircularGauge title usually informs users about the nature of gauge data. To customize it, use the properties of the [title](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/) configuration object:
```

or 

```suggestion
To customize a CircularGauge title, use the properties of the [title](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/) configuration object:
```